test/imported.test.js

```diff
@@ -17,4 +17,5 @@ describe('cloudant imported features', function() {
   require('loopback-datasource-juggler/test/hooks.test.js');
   require('loopback-datasource-juggler/test/relations.test.js');
   require('loopback-datasource-juggler/test/include.test.js');
+  require('loopback-datasource-juggler/test/manipulation.test.js');
```

Comment:
The above is same as require `common.batch.test.js` and `include.test.js`(couchdb2 uses those two instead of listing them all), just since it's extending everything from couchdb2 soon....